test/expected/alternate_users-10.out

```diff
@@ -20,9 +20,9 @@ CREATE INDEX ON PUBLIC."one_Partition" ("timeCustom" DESC NULLS LAST, series_0)
 CREATE INDEX ON PUBLIC."one_Partition" ("timeCustom" DESC NULLS LAST, series_1)  WHERE series_1 IS NOT NULL;
 CREATE INDEX ON PUBLIC."one_Partition" ("timeCustom" DESC NULLS LAST, series_2) WHERE series_2 IS NOT NULL;
 CREATE INDEX ON PUBLIC."one_Partition" ("timeCustom" DESC NULLS LAST, series_bool) WHERE series_bool IS NOT NULL;
-\c single :ROLE_SUPERUSER
+\c :DBNAME :ROLE_SUPERUSER
```

Comment:
What's the difference between `:DBNAME` and `:TEST_DBNAME`?

Reply:
:DBNAME is always the database you are connected to and is updated by psql
:TEST_DBNAME is set by the runner script and does not change during script run

if you connect to template1 in a test then :DBNAME will be template1 while :TEST_DBNAME will still be unchanged

Reply:
in the files that get included with \i I used :DBNAME while in the actual test files i used :TEST_DBNAME this will make included files run in the current database which may be different from :TEST_DBNAME depending on the test